pom.xml

```diff
@@ -121,6 +121,8 @@
     <scala.compat.version>2.10</scala.compat.version>
     <scala.version>2.10.4</scala.version>
     <spark.version>1.3.1</spark.version>
+    <!-- TODO: Remove snapshot dependency when Flink 0.9.1 is released -->
+    <flink.version>0.9-SNAPSHOT</flink.version>
```

Comment:
I also know there were plans to do Mahout DSL support for the shell, but I'm not sure if it got any further.